### PR TITLE
docs(rum-react): lazy rendered routes instrumentation [skip ci]

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ document a solution for elastic/apm-agent-rum-js#487 since there is no easy way to auto instrument the lazy rendered components currently.

### CODE REVIEW

File: docs/react-integration.asciidoc [hunk 1]
```diff
@@ -94,3 +94,35 @@ export default withTransaction('AboutComponent', 'component')(AboutComponent)
 
 NOTE: `withTransaction` accepts two parameters, "transaction name" and "transaction type". 
 If these parameters are not provided, the defaults documented in <<transaction-api, Transaction API>> will be used.
+
+
+[float]
+===== Instrumenting lazy loaded routes
+
+When the route is rendered lazily with components using `React.lazy` or similar API, It is currently not possible to
```
Comment (bmorelli25) [inline docs/react-integration.asciidoc]:
```suggestion
When the route is rendered lazily with components using `React.lazy` or a similar API, it is currently not possible to
```

File: docs/react-integration.asciidoc [hunk 2]
```diff
@@ -94,3 +94,35 @@ export default withTransaction('AboutComponent', 'component')(AboutComponent)
...
+auto instrument the components dependencies via `ApmRoute`. To instrument those lazy rendered routes and capture the spans
```
Comment (bmorelli25) [inline docs/react-integration.asciidoc]:
```suggestion
auto instrument the components dependencies via `ApmRoute`. To instrument these lazy rendered routes and capture the spans 
```

File: docs/react-integration.asciidoc [hunk 3]
```diff
@@ -94,3 +94,35 @@ export default withTransaction('AboutComponent', 'component')(AboutComponent)
...
 NOTE: `withTransaction` accepts two parameters, "transaction name" and "transaction type". 
 If these parameters are not provided, the defaults documented in <<transaction-api, Transaction API>> will be used.
+
+
+[float]
+===== Instrumenting lazy loaded routes
+
+When the route is rendered lazily with components using `React.lazy` or similar API, It is currently not possible to
+auto instrument the components dependencies via `ApmRoute`. To instrument those lazy rendered routes and capture the spans 
+associated with the components, The user would need to manually instrument the code via `withTransaction` API.
```
Comment (bmorelli25) [inline docs/react-integration.asciidoc]:
```suggestion
associated with the components, you'll need to manually instrument the code with the `withTransaction` API.
```

**Comment by vigneshshanmugam** [comment]
Thanks @bmorelli25

**Pull request merged**